src/Files.App/Helpers/ShellContextMenuHelper.cs

```diff
@@ -264,9 +264,10 @@ async Task InvokeShellMenuItem(ContextMenu contextMenu, object? tag)
 				if (openWithItem is not null)
 					shellMenuItems.Remove(openWithItem);
 
-				var sendToItem = showSendToMenu ? shellMenuItems.Where(x => (x.Tag as Win32ContextMenuItem)?.CommandString == "sendto").ToList().FirstOrDefault() : null;
-				if (sendToItem is not null)
+				var sendToItem = shellMenuItems.Where(x => (x.Tag as Win32ContextMenuItem)?.CommandString == "sendto").ToList().FirstOrDefault();
+				if (sendToItem is not null && (showSendToMenu || !UserSettingsService.GeneralSettingsService.ShowSendToMenu))
 					shellMenuItems.Remove(sendToItem);
```

Comment:
Can you split the statement into multiple lines?

Reply:
Updated. Is this ok?

Reply:
Thanks